pyproject.toml

```diff
@@ -22,8 +22,11 @@ classifiers = [
 dynamic = ["version"]
 dependencies = ["pydantic"]
 
+[project.entry-points]
+hypothesis = {_ = "geojson_pydantic.strategies:_hypothesis_setup_hook"}
+
 [project.optional-dependencies]
-test = ["pytest", "pytest-cov", "shapely"]
+test = ["pytest", "pytest-cov", "shapely", "hypothesis"]
```

Comment:
I guess `hypothesis` becomes at least an optional dependency of the whole module

Reply:
None of the code needed for the main module imports from `strategies` so it doesn't require `hypothesis` to run. It's an optional part that can be used if you're using hypothesis already. So, I'd say it's not something to include in the main requirements. And `pydantic` itself actually has a plugin built in and doesn't include it in it's dependents at all.

Reply:
👍 I'm not sure how to advertise that there is an optional dependency. 

it seems kinda weird to add `hypothesis = ["hypothesis"]` but it might be the best solution 🤷

Reply:
❤️